### PR TITLE
Updated Kotlin

### PR DESCRIPTION
**Pull request by gcantoni**
Signed-off-by: Giorgio Cantoni <giorgio.canto98@gmail.com>

### CODE REVIEW

**Comment by gcantoni** [comment]
@gcantoni @svandasek

**Comment by jansvanda** [comment]
Merge pls, will save noobs like me time

**Comment by msfjarvis** [comment]
@jansvanda ?
@gcantoni We follow AS Canary, which ships plugin 1.2.71

**Comment by gcantoni** [comment]
@MSF-Jarvis as @jansvanda said in group he got an error during building the theme related to the gradle kotlin version. It will be helpful to other themers facing this issue.

File: build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.2.50'
+    ext.kotlin_version = '1.2.51'
```
Comment (msfjarvis) [inline build.gradle]: 1.2.71

**Review by msfjarvis — CHANGES_REQUESTED** [review]

**Comment by gcantoni** [comment]
@MSF-Jarvis merge 😄

**Pull request merged**